nengo/builder/operator.py

```diff
@@ -643,6 +643,70 @@ def step_dotinc():
         return step_dotinc
 
 
+class BsrDotInc(Operator):
```

Comment:
Maybe note the usage of this operator, like you do in your report? Also, how likely is it that a model won't require this operation if it's being optimized? Because, if a person tries to turn on the optimizer, fails the import, it would be nice to send them a warning informing them that if they want an actual performance boost they should use SciPy. Similar to what we do with the Basal Ganglia.